app/src/main/java/org/fossasia/openevent/api/processor/EventListResponseProcessor.java

```diff
@@ -0,0 +1,25 @@
+package org.fossasia.openevent.api.processor;
+
+import android.util.Log;
+
+import org.fossasia.openevent.api.protocol.EventResponseList;
+
+import retrofit.Callback;
+import retrofit.RetrofitError;
+import retrofit.client.Response;
+
+/**
+ * Created by MananWason on 27-05-2015.
+ */
+public class EventListResponseProcessor implements Callback<EventResponseList> {
+    private static final String TAG = "Events";
+    @Override
+    public void success(EventResponseList eventResponseList, Response response) {
+        Log.d(TAG," done");
```

Comment:
You can check out https://github.com/JakeWharton/timber for Log related stuff. `done` within the Log message means nothing, It should be used only for development.